### PR TITLE
refactor to agent deploy scenario reduce run time

### PR DESCRIPTION
**Pull request by EricDavisX**
## What does this PR do?

This is a small refactor that I'm pulling out of an older PR we are not likely to follow up on.  see comment here: https://github.com/elastic/e2e-testing/pull/178#discussion_r456400209

This will make the tests run a tiny bit faster, skipping a basically redundant scenario.

I plan to have the test run validate it works.  ;/  I'll check the boxes shortly or strike them out.

## Checklist
- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run the Unit tests for the CLI, and they are passing locally
- [ ] I have run the End-2-End tests for the suite I'm working on, and they are passing locally
- [ ] I have noticed new Go dependencies (run `make notice` in the proper directory)

### CODE REVIEW

File: e2e/_suites/ingest-manager/features/fleet_mode_agent.feature [hunk 1]
```diff
@@ -6,17 +6,9 @@ Feature: Fleet Mode Agent
...
-Examples:
-| os     |
-| centos |
-| debian |
-
-@start-agent
-Scenario Outline: Starting the <os> agent starts backend processes
-  When a "<os>" agent is deployed to Fleet
-  Then the "filebeat" process is in the "started" state on the host
+    And the "filebeat" process is in the "started" state on the host
```
Comment (mdelapenya) [inline e2e/_suites/ingest-manager/features/fleet_mode_agent.feature:9]: What do you think of adding `And the "elastic-agent" process is in the "started" state on the host` as first check?

Comment (EricDavisX) [reply e2e/_suites/ingest-manager/features/fleet_mode_agent.feature:10]: love it. good idea!

**Review by mdelapenya — APPROVED** [review]
LGTM 👍 

image

File: e2e/_suites/ingest-manager/features/fleet_mode_agent.feature (hunk 1 above)
Comment (mdelapenya) [reply e2e/_suites/ingest-manager/features/fleet_mode_agent.feature:10]: Super NIT: What about reordering the steps in terms of importance or order of appearance?

```gherkin
Then the "elastic-agent" process is in the "started" state on the host
    And the "filebeat" process is in the "started" state on the host
    And the "metricbeat" process is in the "started" state on the host
    And the agent is listed in Fleet as "online"
    And system package dashboards are listed in Fleet
```

Comment (mdelapenya) [reply e2e/_suites/ingest-manager/features/fleet_mode_agent.feature:10]: @EricDavisX; i'm gonna merge and send a PR with the comments above.

Thanks for your contribution!!

**Pull request merged**